book_source/03_intermediate_users_guide/04_remote_execution.Rmd

```diff
@@ -266,20 +273,20 @@ are additional entries to add to the job.sh file generated to
 run the model. This can be used to make sure modules are loaded
 on the HPC cluster before running the actual model.
 
-## Running PEcAn code for modules remotely
+## Running PEcAn code for remotely
 
-You can compile and install the model specific code pieces of
+You do not need to download PEcAn fully on your remote machine. You can compile and install the model specific code pieces of
 PEcAn on the cluster easily without having to install the
 full code base of PEcAn (and all OS dependencies). All of the
 code pieces depend on PEcAn.utils to install this you can
-run the following on the cluster:
+run the following on the remote machine:
 
 ```
 devtools::install_github("pecanproject/pecan", subdir = 'utils')
```

Comment:
1) `utils` is in `base`. I suspect this documentation hasn't been updated since we created `base`
2) I wouldn't start with `utils` but would rather start with an example of what one actually wants to do on remote (e.g. lets say you want to run the model ED2) and then show how to check what other PEcAn packages that depends on, https://github.com/PecanProject/pecan/blob/develop/models/ed/DESCRIPTION#L16-L21,  and then give the example of grabbing those before grabbing ED2.
3) that said, this may get you into an annoying pattern of those packages having dependencies. This suggest two things to me. First, that  we need to **first** show the alternative way to do this (git clone the whole repo but only make the specific package you need, and then `make` will sort out what other pecan packages that depends on. Second, it might not be bad for somewhere in the documentation to have a figure of pecan package dependencies  (just pecan, not CRAN). That said, I don't think such a figure needs to be in this PR. I also worry that unless there's a pre-existing elegant solution to do this automatically we'll either get stuck in a rabbithole of figuring out how to do this via code or doing it by hand and having it forever be out-of-date.